### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by MatthewOakley**
I did some slight work on the admin prototype. I am breaking up the work into multiple sections and doing a little bit of work each day. I had some trouble pulling with index.jsp but I believed I fixed it.

### CODE REVIEW

File: src/main/java/codeu/controller/AdminServlet.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.jsoup.safety.Whitelist;
+
+/**
+ * AdminServlet 
+ *
+ * This servlet class is for the admin page
+ */
+public class AdminServlet extends HttpServlet {
+  
+  // setup the admin page
```
Comment (erschwartz) [inline src/main/java/codeu/controller/AdminServlet.java]: Can you do your comments Javadoc style please? https://google.github.io/styleguide/javaguide.html#s7-javadoc

File: src/main/java/codeu/controller/AdminServlet.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Admin Servlet
```
Comment (erschwartz) [inline src/main/java/codeu/controller/AdminServlet.java]: This is unnecessary, you can delete

File: src/main/webapp/WEB-INF/view/admin.jsp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
```
Comment (erschwartz) [inline src/main/webapp/WEB-INF/view/admin.jsp]: I'd write a TO-DO comment here to say that you'll be replacing this later on.
We usually format these:
`//TO-DO($USERNAME) Some comment here`

**Review by erschwartz — CHANGES_REQUESTED** [review]

**Comment by erschwartz** [comment]
Small PR, good first start!

**Comment by MatthewOakley** [comment]
I fixed the styling. Let me know if I am doing the JavaDocs styling wrong. I also added more To-Do's where I will be adding more code later.

File: src/main/java/codeu/controller/AdminServlet.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ * and showing stats about it.
+ */
+public class AdminServlet extends HttpServlet {
+  /** 
+   * TO-DO(Matthew Oakley) I need to add getting the data from the 
+   * the database and checking to make sure its valid and not null
+   * and sending this information to the webpage
+   */
+  
+  // setup the admin page
```
Comment (erschwartz) [inline src/main/java/codeu/controller/AdminServlet.java]: Docstring style here please

File: src/main/java/codeu/controller/AdminServlet.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  @Override
+  public void init() throws ServletException {
+    super.init();
+  }
+  
+  // when user visits sends them to admin page
```
Comment (erschwartz) [inline src/main/java/codeu/controller/AdminServlet.java]: Docstring style

always use this style for function comments and class comments :)

**Review by erschwartz — APPROVED** [review]
After you fix these two, LGTM!

**Comment by erschwartz** [comment]
@StevenAb please try to review this ASAP!

File: src/main/java/codeu/controller/AdminServlet.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  public void init() throws ServletException {
+    super.init();
+  }
+  
+  /** when user visits this sends them to admin page */
+  @Override
+  public void doGet(HttpServletRequest request, 
+      HttpServletResponse response)
+      throws IOException, ServletException {
+    String requestUrl = request.getRequestURI();
```
Comment (StevenAb) [inline src/main/java/codeu/controller/AdminServlet.java:37]: What is the string 'requestUrl' used for?

File: src/main/webapp/WEB-INF/view/admin.jsp [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+</head>
+<body>
+  <!-- 
+    TO-DO(Matthew Oakley) I am going to add the stats for the admin page
+    I just have this basic setup to see if I could get the servlets working
+  -->
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null) { %>
```
Comment (StevenAb) [inline src/main/webapp/WEB-INF/view/admin.jsp:18]: Does the user check still work without setting the UserStore in the servlet?

**Review by StevenAb — APPROVED** [review]
Had two minor questions, but overall LGTM!

File: src/main/java/codeu/controller/AdminServlet.java (hunk 6 above)
Comment (MatthewOakley) [reply src/main/java/codeu/controller/AdminServlet.java:37]: This is if I need it later on for any type of displaying I want to do with the url.

File: src/main/webapp/WEB-INF/view/admin.jsp (hunk 7 above)
Comment (MatthewOakley) [reply src/main/webapp/WEB-INF/view/admin.jsp:18]: Yes.

**Pull request merged**